articles/support/pre-deployment/how-to-run-test.md

```diff
@@ -0,0 +1,56 @@
+---
+description: How to run the Auth0 Pre-Deployment Tests to ensure that your Clients are production-ready
+---
+
+# How to Run the Pre-Deployment Test
+
+The Pre-Deployment Tests are available via the [Auth0 Support Center](https://support.auth0.com/).
```

Comment:
@khorne3 I probably made the wrong assumption then. Since only paid and trial customer can log tickets through the Support Center I assumed that only they have access to all Support Center functionality.